pkg/smokescreen/config_loader.go

```diff
@@ -48,71 +48,71 @@ type yamlConfig struct {
 	Tls *yamlConfigTls
 	// Currently not configurable via YAML: RoleFromRequest, Log, DisabledAclPolicyActions
 
-	UnsafeAllowPrivateRanges bool	`yaml:"unsafe_allow_private_ranges"`
+	UnsafeAllowPrivateRanges bool `yaml:"unsafe_allow_private_ranges"`
 }
 
-func (c *Config) UnmarshalYAML(unmarshal func(interface{}) error) error {
+func (config *Config) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	var yc yamlConfig
-	*c = *NewConfig()
+	*config = *NewConfig()
```

Comment:
Single letter variable usage seem to be a stylistic choice here, seems like minimizing the amount of changes might results in the PR being more likely to be merged, WDYT?